mne/io/bci/bci.py

```diff
@@ -0,0 +1,178 @@
+"""Conversion tool from OpenBCI to FIF"""
+
+# Authors: Teon Brooks <teon.brooks@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from ...utils import verbose
+from ..meas_info import create_info
+from ..base import _BaseRaw
+
+
+class RawBCI(_BaseRaw):
+    """Raw object from BCI file
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the BCI file.
+    montage : str | None | instance of Montage
+        Path or instance of montage containing electrode positions.
+        If None, sensor locations are (0,0,0). See the documentation of
+        :func:`mne.channels.read_montage` for more information.
+    eog : list or tuple
+        Names of channels or list of indices that should be designated
+        EOG channels. Default is None.
+    misc : list or tuple
+        List of indices that should be designated MISC channels.
+        Default is (-3, -2, -1), which are the accelerator sensors.
+    reference : None | str
+        Name of the electrode which served as the reference in the recording.
+        If a name is provided, a corresponding channel is added and its data
+        is set to 0. This is useful for later re-referencing. The name should
+        correspond to a name in elp_names. Data must be preloaded.
+    scale : float
+        The scaling factor for EEG data. Units for MNE are in volts.
+        BCI data are typically stored in microvolts. Default scale
+        factor is 1e-6.
+    sfreq : int
+        The sampling frequency of the data. BCI defaults are 250 Hz.
+    preload : bool
+        If True, all data are loaded at initialization.
+        If False, data are not read until save.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Notes
+    -----
+    .. versionadded:: 0.10.0
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    @verbose
+    def __init__(self, input_fname, montage=None, eog=None,
+                 misc=(-3, -2, -1), reference=None, scale=1e-6, sfreq=250,
+                 preload=True, verbose=None):
+
+        if not eog:
+            eog = list()
```

Comment:
FYI a one-line equivalent is `eog = eog or list()` but it's probably too obfuscated for use here